content/03.methods.md

```diff
@@ -44,7 +44,9 @@ We used Control-FREEC [@doi:10/ckt4vz; @doi:10/c6bcps] v8.7 for copy number vari
 ### Somatic Structural Variant Calling
 
 We used Manta SV [@doi:10/gf3ggb] v1.4.0 for structural variant (SV) calls.
-SV calling was also limited to regions using in Strelka2.
+Manta SV calling was also limited to regions using in Strelka2.
+LUMPY SV [@doi:10/gf3ggc] v0.2.13 was also used for SV calls.
```

Comment:
Perhaps reword a bit to: ```We also ran LUMPY SV [@doi:10/gf3ggc] v0.2.13 in express mode using default parameters.
The hg38 reference used was also limited to canonical chromosome regions..```

Reply:
I like that rewording, it flows better than what I had. I made the changes to the file.